pandas/tests/io/sas/data/DRXFCD_G.csv

```diff
@@ -4551,7 +4551,7 @@
 58101955,"TACO OR TOSTADA SALAD, MEATLESS W/ SOUR CREAM","Taco or tostada salad, meatless with sour cream"
 58103120,"TAMALE WITH MEAT","Tamale with meat"
 58103130,"TAMALE WITH CHICKEN","Tamale with chicken"
-58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Carribean Style"
+58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Caribbean Style"
```

Comment:
```suggestion
58103200,"TAMALE, PLAIN, MEATLESS, NO SAUCE, PR STYLE","Tamale, plain, meatless, no sauce, Puerto Rican style or Carribean Style"
```
I guess this is for tests with SAS, that need to march a SAS file with a typo. Let's revert here, and feel free to fix it in a separate PR for here, and for the SAS file.

Reply:
let me know if is ok